USAGE.md

```diff
@@ -4572,7 +4572,7 @@ print response.headers
 
 **This endpoint allows you to retrieve the current credit balance for your account.**
 
-Your monthly credit allotment limits the number of emails you may send before incurring overage charges. For more information about credits and billing, please visit our [Classroom](https://sendgrid.com/docs/Classroom/Basics/Billing/billing_info_and_faqs.html).
+Your monthly credit allotment limits the number of emails you may send before incurring coverage charges. For more information about credits and billing, please visit our [Classroom](https://sendgrid.com/docs/Classroom/Basics/Billing/billing_info_and_faqs.html).
```

Comment:
'overage' is correct in this context.

Reply:
Spelling fixed! I didn't know that word, got a litle over-zealous on the corrections. I wasn't 100% sure how to tackle rebasing a PR but I've tried doing a PR against V4.